src/OSPSuite.Infrastructure.Import/Services/IImportFromExcelTask.cs

```diff
@@ -100,13 +100,23 @@ private static IEnumerable<DataTable> dataTables(string fileName, string sheetNa
 
             while (reader.MoveToNextRow())
             {
-               dataTable.AddRowToDataTable(reader.CurrentRow);
+               var rowList = reader.CurrentRow;
```

Comment:
i am surprised about this naming rowList...when you have one row?

Reply:
isn't it more rowValues?

Reply:
true. corrected